src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
 
+use Prophecy\Argument;
```

Comment:
Do we need to use Prophecy here? And if so we need to add a dependency to `composer.json`.